tf_quant_finance/examples/jupyter_notebooks/Swap_Curve_Fitting.ipynb

```diff
@@ -0,0 +1,533 @@
+    "fixed_leg_daycounts = []\n",
+    "fixed_leg_cashflows = []\n",
+    "pvs = []\n",
+    "initial_curve_rates = []\n",
+    "\n",
+    "instruments = [\n",
+    "    build_swap_instrument(start=0, tenor=1, fixed_payment_freq=2, float_payment_freq=4, rate=0.014114),\n",
+    "    build_swap_instrument(start=0, tenor=2, fixed_payment_freq=2, float_payment_freq=4, rate=0.012476),\n",
+    "    build_swap_instrument(start=0, tenor=3, fixed_payment_freq=2, float_payment_freq=4, rate=0.011860),\n",
+    "    build_swap_instrument(start=0, tenor=4, fixed_payment_freq=2, float_payment_freq=4, rate=0.011695),\n",
+    "    build_swap_instrument(start=0, tenor=5, fixed_payment_freq=2, float_payment_freq=4, rate=0.011725),\n",
+    "    build_swap_instrument(start=0, tenor=6, fixed_payment_freq=2, float_payment_freq=4, rate=0.011892),\n",
+    "    build_swap_instrument(start=0, tenor=7, fixed_payment_freq=2, float_payment_freq=4, rate=0.012120),\n",
+    "    build_swap_instrument(start=0, tenor=8, fixed_payment_freq=2, float_payment_freq=4, rate=0.012384),\n",
+    "    build_swap_instrument(start=0, tenor=9, fixed_payment_freq=2, float_payment_freq=4, rate=0.012674),\n",
+    "    build_swap_instrument(start=0, tenor=10, fixed_payment_freq=2, float_payment_freq=4, rate=0.012975),\n",
+    "    build_swap_instrument(start=0, tenor=12, fixed_payment_freq=2, float_payment_freq=4, rate=0.013545),\n",
+    "    build_swap_instrument(start=0, tenor=15, fixed_payment_freq=2, float_payment_freq=4, rate=0.014172),\n",
+    "    build_swap_instrument(start=0, tenor=20, fixed_payment_freq=2, float_payment_freq=4, rate=0.014855),\n",
+    "    build_swap_instrument(start=0, tenor=25, fixed_payment_freq=2, float_payment_freq=4, rate=0.015075),\n",
+    "    build_swap_instrument(start=0, tenor=30, fixed_payment_freq=2, float_payment_freq=4, rate=0.015096),\n",
+    "    build_swap_instrument(start=0, tenor=35, fixed_payment_freq=2, float_payment_freq=4, rate=0.014916),    \n",
+    "]\n",
+    "                      \n",
+    "for instrument in instruments:\n",
+    "    float_leg_start_times.append(instrument['float_leg_start_time'])\n",
+    "    float_leg_end_times.append(instrument['float_leg_end_time'])\n",
+    "    float_leg_daycounts.append(instrument['float_leg_daycount'])\n",
+    "    fixed_leg_start_times.append(instrument['fixed_leg_start_time'])\n",
+    "    fixed_leg_end_times.append(instrument['fixed_leg_end_time'])\n",
+    "    fixed_leg_daycounts.append(instrument['fixed_leg_daycount'])\n",
+    "    fixed_leg_cashflows.append(instrument['fixed_leg_cashflow'])\n",
+    "    pvs.append(instrument['pv'])\n",
+    "    initial_curve_rates.append(instrument['initial_curve_rate'])\n",
+    "    \n",
+    "curve = swap_curve_fit(\n",
+    "    float_leg_start_times,\n",
+    "    float_leg_end_times,\n",
+    "    float_leg_daycounts,\n",
+    "    fixed_leg_start_times,\n",
+    "    fixed_leg_end_times,\n",
+    "    fixed_leg_cashflows,\n",
```

Comment:
Could you please pass all the arguments by name as opposed to positions `(float_leg_start_times=float_leg_start_times, ...)`? The library is in dev stage, so this will safeguard from changing the function signature.

Reply:
Named parameters addressed in c537ef8b7d6946faeb832c2a90a18d0eab5d38f0